react/Autosuggest/Autosuggest.demo.js

```diff
@@ -94,6 +94,22 @@ export default {
             className: classnames(className, styles.rootFocus)
           })
         },
+        {
+          label: 'Grouped',
+          transformProps: ({ autosuggestProps, ...props }) => ({
+            ...props,
+            autosuggestProps: {
+              ...autosuggestProps,
+              multiSection: true,
```

Comment:
Realised that I had no way to preview the updated section heading colour, so took this as a good opportunity to add this to our demo page 😄🙌